### ISSUE TITLE
UI Feature requests (Low)

### ISSUE THREAD

**Issue by Chaney08** [PolyPup-Farm/polypup-ui-alpha-testers#19]
When the toast menu pops up that a transaction has completed, it would be nice if I could click on that to bring me to the polygon transaction.

Another small UI request, the "Show Staked only" button could do with a bit more clarity, it is a small radio button off to the side, it would be nice if that was a proper button that was a bit more obvious (Very minor as I still found it easily)

**Comment by Polypup** [comment]
Yes, I will be adding a small section for recent transactions. The toast might be a bit too fast to be clickable.
